src/fa/game_session.py

```diff
@@ -5,7 +5,7 @@
 from connectivity.turn import TURNState
 from decorators import with_logger
 from fa.game_connection import GPGNetConnection
-from fa.game_process import GameProcess, instance as game_process_instance
```

Comment:
My linter complains that GameProcess is unused, might fix that too while we're at it.

Reply:
and your linter is right, pycharm was fooled by the type comment in line 67

Reply:
and while we're at it .. what does your linter say about:
'import chat'  and  'import sys' in _clientwindow.py?

Reply:
It complains about these and a couple more lines. I'd rather avoid mass pruning these just yet to make rebasing the ice branch easier, this one is just in the way anyway. It'd be trivial to do either way, I'm using flake8 as a linter and I'm sure there are ways to automatically prune the stuff.

Reply:
still complains here, after last update?

Reply:
pycharm is unhappy with the 3  @_needs_game_connection
it would be happy with @_needs_game_connection()
is that a thing, or just stupid pycharm?